### ISSUE TITLE
BUG: Ensure test line for Kasa Na Tua is always available, no busy tones

### ISSUE THREAD

**Issue by jemymaRC** [jemymaRC/KasaNaTua#7]
Currently, the test line for Kasa Na Tua gets a busy tone hindering testing.

Please fix to ensure constant access.

**Comment by jemymaRC** [comment]
@tektrix1, @dlpgh: I suspect that the busy tone on the Kasa Na Tua test line occurs after the system cuts the line automatically when the user gets to the end of their request tree path. 

I say this because I called the number a few minutes ago to retest Making a deposit for self option and went all the way to the last menu option which is to indicate which network my number is associated with (after entering the correct, or even wrong, network it tells me to call back again with the correct network and hangs up). I then called the line again to start testing Make a deposit for a third party and the line is now busy.

Could this be a coding issue? Maybe even though the line cuts it is not terminated properly in the system hence the busy tone (so even though my phone call cuts, in the system it hasn't).
